syntax="proto3";
// 指定模板类的包路径
option java_package = "com.szlongzy.common.protobuf.dto";
// 指定模板类的名称，名称必须是有实际业务意义的
option java_outer_classname = "FirmwareVerVo";


/* 获取版本请求*/
message VersionRequest{

  // 实际目标设备
  string  objDevId = 1;
  //备用
  string authKey =2;
}

/* 版本信息上报*/
message VersionResponse{
  // 回复
  bool  result = 1;
  // 实际目标设备
  string  objDevId = 2;
  // 错误信息
  string  msg = 3;
  string authKey =4;
  FirmwareVersionVo data = 5;
}

//固件版本信息
message FirmwareVersionVo{
  //通用固件版本
  string fmvVer = 1;
  //sco固件版本
  string socVer = 2;
  //mcu固件版本
  string mcuVer = 3;
  //无线固件版本
  string wirelessVer = 4;
  //备用字段
  string standby = 5;

}

